appveyor.yml

```diff
@@ -35,8 +35,7 @@ install:
 
     # Add path, activate `conda` and update conda.
     - cmd: call %CONDA_INSTALL_LOCN%\Scripts\activate.bat
-    - cmd: conda update --yes --quiet conda
-    - cmd: conda config --system --add pinned_packages defaults::conda
+    - cmd: conda update --yes --quiet conda=4.3
```

Comment:
`conda install` instead of `conda update`

Reply:
Thanks, @isuruf. Just pushed the change.